components/scream/src/interface/scream_cxx_f90_interface.cpp

```diff
@@ -1,57 +1,125 @@
-#include "interface/ScreamContext.hpp"
+#include <catch2/catch.hpp>
```

Comment:
I don't think this catch include is needed.

Reply:
I'll remove it and see what happens.